runbotics/runbotics-orchestrator-ui/src/views/process/ProcessConfigureView/ProcessTriggerableComponent.tsx

```diff
@@ -14,18 +14,23 @@ interface ProcessTriggerableProps {
 
 const ProcessTriggerableComponent: VFC<ProcessTriggerableProps> = ({ isProcessTriggerable, onTriggerableChange }) => {
     const hasReadIsProcessTriggerable = useFeatureKey([FeatureKey.PROCESS_IS_TRIGGERABLE_READ]);
+    const hasEditIsProcessTriggerable = useFeatureKey([FeatureKey.PROCESS_IS_TRIGGERABLE_EDIT]);
```

Comment:
these consts do not look good, let's rephrase them in a way:
`hasEditProcessTriggerAccess` and `hasReadProcessTriggerAccess`

do the same in a attended component